org.leocad.LeoCAD.json

```diff
@@ -35,7 +35,8 @@
             ],
             "post-install": [
                 "mkdir -p /app/share/icons/hicolor/scalable/apps",
-                "cp /app/share/icons/hicolor/scalable/mimetypes/application-vnd.leocad.svg /app/share/icons/hicolor/scalable/apps/leocad.svg"
+                "cp /app/share/icons/hicolor/scalable/mimetypes/application-vnd.leocad.svg /app/share/icons/hicolor/scalable/apps/leocad.svg",
```

Comment:
You'll have to rename this to be exported.

Reply:
1. it is renamed with `rename-icon`
2. it's not the subject of this PR. The line if modified because JSON doesn't like trailing `,`

Reply:
If I understand what you've done correctly, the empty categories are expected because the app is using the default categories for the complete library instead of the reduced category set for the fallback library.

What are you trying to fix with this change? The library is bundled with the executable already.

Reply:
It show that dialog when I start LeoCAD
![Screenshot from 2020-02-11 20-14-27](https://user-images.githubusercontent.com/114441/74294293-63a5a300-4d0b-11ea-88d3-4f3ccf25f53f.png)

Reply:
Actually I see now. We need to copy the library from https://www.ldraw.org/parts/latest-parts.html

The problem with that is it seems that ldraw doesn't have a version name in the archive. Which mean that at the next update the build will fail.